src/tracing/trace.test.ts

```diff
@@ -32,6 +32,7 @@ describe("createTracerProvider", () => {
       subject = createTracerProvider(
         "localhost",
         "test=foo",
+        "",
```

Comment:
We can remove all the empty strings in the tests after making the change non-breaking.